staker-rewards/package.json

```diff
@@ -11,8 +11,8 @@
     "test": "NODE_ENV=test node --experimental-json-modules node_modules/mocha/bin/mocha",
     "lint": "npm run lint:js",
     "lint:fix": "npm run lint:fix:js",
-    "lint:js": "eslint .",
-    "lint:fix:js": "eslint --fix ."
+    "lint:js": "eslint ${npm_package_config_eslintPaths}",
+    "lint:fix:js": "eslint --fix ${npm_package_config_eslintPaths}"
```

Comment:
No, we don't need this change, dosn't `eslint .` work?

Reply:
Yup, it works, done in a3ab8b7

Reply:
a3ab8b7 updates `solidity/package.json`, we still need to update `staker-rewards/package.json`.

Reply:
a5f1e04